pkgs/secp256k1/default.nix

```diff
@@ -1,26 +1,47 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+}:
 
 stdenv.mkDerivation {
   pname = "secp256k1";
 
-  version = "2019-10-11";
+  version = "2021-12-03";
 
   src = fetchFromGitHub {
     owner = "bitcoin-core";
     repo = "secp256k1";
-    rev = "0d9540b13ffcd7cd44cc361b8744b93d88aa76ba";
-    sha256 = "05zwhv8ffzrfdzqbsb4zm4kjdbjxqy5jh9r83fic0qpk2mkvc2i2";
+    rev = "490022745164b56439688b0fc04f9bd43578e5c3";
+    hash = "sha256-6CmGWiecthaGWSKX7VHWj5zvDAwVE9U5YOo9JRJWYwI=";
   };
 
   nativeBuildInputs = [ autoreconfHook ];
 
-  configureFlags = ["--enable-module-recovery" "--disable-jni" "--enable-experimental" "--enable-module-ecdh" "--enable-benchmark=no" ];
+  configureFlags = [
+    "--enable-benchmark=no"
+    "--enable-exhaustive-tests=no"
+    "--enable-experimental"
+    "--enable-module-ecdh"
+    "--enable-module-recovery"
+    "--enable-module-schnorrsig"
+    "--enable-tests=yes"
```

Comment:
nit: tests are enabled by default

Reply:
Fixed in 6629e9a66f1b4aea5feb30a89c0be0ff5c6dd1a0